ledger/sandbox/README.md

```diff
@@ -45,3 +45,13 @@ Note that the new Ledger API only supports DAML 1.0 or above codebases compiled
 # Ledger API
 
 The new Ledger API uses gRPC. You can find the full documentation of all the services involved rendered at http://ci.da-int.net/job/ledger-api/job/build/job/master/lastSuccessfulBuild/artifact/ledger-api/grpc-definitions/target/docs/index.html (save the file locally to get the styling to work). If you just want to create / exercise contracts, I suggest you start by looking at `command_service.proto`, which exposes a synchronous API to the DAML ledger. 
+
+# Logging
+
+You can enable debug logging in Sandbox with `sandbox-log-level` system property:
+
+    $ java -Dsandbox-log-level=DEBUG -jar ./bazel-bin/ledger/sandbox/sandbox-binary_deploy.jar $PWD/bazel-bin/ledger/sandbox/Test.dar
```

Comment:
System parameters don't play nice with the daml assistant, as far as I know. @associahedron, is this correct?

Reply:
AFAIK there's no way to set them rwhen running via `daml`. Both command-line options and environment variables would work though.

Reply:
My suggestion: use an environment variable `SANDBOX_LOG_LEVEL` or so.

Reply:
@gerolf-da Any ideas how to thread through an env variable `$SANDBOX_LOG_LEVEL` into logback configuration? Happy to do it if I get a hint how to

Reply:
You could also do both -- if `-Dsandbox-log-level` is set, use that, otherwise use `SANDBOX_LOG_LEVEL`?

Reply:
I would rather make it a regular command line option. This mechanism is known to work. You can set system properties prior to initializing the logging system.

Reply:
There are 2 examples on how to set the log level programmatically in [Java Codegen](https://github.com/digital-asset/daml/blob/master/language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/CodeGenRunner.scala#L32-L39) and [Scala Codegen](https://github.com/digital-asset/daml/blob/master/language-support/scala/codegen/src/main/scala/com/digitalasset/codegen/Main.scala#L33-L41)

Reply:
Done